web/src/main/java/it/pagopa/selfcare/dashboard/web/model/EmbeddedExternalIdDto.java

```diff
@@ -0,0 +1,11 @@
+package it.pagopa.selfcare.dashboard.web.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
```

Comment:
@KevinSi96 use `@Data` instead of `@Getter` and `@Setter` so Lombok will create the `toString()` method too